src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -49,6 +49,7 @@ import ChartbeatAnalytics from '../../components/ChartbeatAnalytics';
 import LinkedData from '../../components/LinkedData';
 import Uploader from '../../components/Uploader';
 import Byline from '../../components/Byline';
+import VJLoader from '../../components/VJLoader';
```

Comment:
we going to have an issue when we implement actual VJ includes later? Think they're going to be done a different way to these flourish embeds?

Also, the ticket from the branch name points to riddle embeds, so not sure which ticket this is now.